rename_wpt/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/google/go-github/v35/github"
+	"github.com/web-platform-tests/wpt-metadata/go_util"
+)
+
+// This main() gets all the WPT test renaming and deletions between two WPT SHAs, then it deletes/renames
+// appropriately in the wpt-metadata repository. This program will roll in the latest WPT SHA once a day
+// through a cron job.
+func main() {
+	// TODO(Kyleju): plumb the before, after SHA through shell. Update SHA through shell as well.
+	shaAfter := "e021a4c8f92596f3261e2d7540d017f5138deb68"
+	shaBefore := "012651bc39bb45a6dc9c93a968b47dd9310a1e8a"
+
+	// Plumb the SHA information from a text file, WPT SHA rolls on a daily basis
+	renames, deletes := getChangesBetweenSHAs(context.Background(), shaBefore, shaAfter)
+	if renames == nil {
+		return
+	}
+
+	fmt.Println("Renaming WPT tests...")
+	for before, after := range renames {
+		renameMetadata(before, after)
```

Comment:
`getChangesBetweenSHAs` gets the renamed files, right? This should mean that `.any.js` tests aren't handled, since in the metadata we use test names. Not sure how much effort to spend on this, but maybe a TODO acknowledging the limitation would be good?

Reply:
Also discussed this in our 1:1. I will leave a TODO